grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -610,12 +640,17 @@ public GrpcService build() {
         if (USE_COROUTINE_CONTEXT_INTERCEPTOR) {
             final ServerInterceptor coroutineContextInterceptor =
                     new ArmeriaCoroutineContextInterceptor(useBlockingTaskExecutor);
+            final List<ServerInterceptor> interceptors = new ArrayList<>();
+            if (this.interceptors != null) {
+                interceptors.addAll(this.interceptors.build());
+            }
+            interceptors.add(coroutineContextInterceptor);
```

Comment:
Don't need to create addtional `List`.
```suggestion
            interceptors.add(coroutineContextInterceptor);
```